lib/src/clojure_lsp/feature/resolve_macro.clj

```diff
@@ -57,7 +57,8 @@
              (update-macro-resolve-for-config (symbol resolved-full-symbol-str) full-symbol))))))
 
 (def ^:private known-full-symbol-resolve
-  ["clojure.core/def"
+  ["clojure.core/fn"
```

Comment:
I think lint as fn is less common than lint as def or defn or let, could you move it to below `clojure.core/let`?

Reply:
Oh, sorry, I forgot about that earlier! Fixed.